tool/tctl/common/collection.go

```diff
@@ -715,14 +723,18 @@ func (c *windowsDesktopCollection) resources() (r []types.Resource) {
 }
 
 func (c *windowsDesktopCollection) writeText(w io.Writer) error {
-	t := asciitable.MakeTable([]string{"UUID", "Address"})
+	t := asciitable.MakeTable([]string{"Host", "Public Address", "AD Domain", "Labels", "Version"})
 	for _, desktop := range c.desktops {
-		t.AddRow([]string{desktop.GetName(), desktop.GetAddr()})
+		t.AddRow([]string{"", desktop.GetAddr(), desktop.GetDomain(), desktop.LabelsString()[:10], ""})
```

Comment:
Maybe consider hiding any labels with a `teleport.dev/` prefix? Those are the ones that are auto-populated via LDAP and likely not important to something browsing via `tctl`.